libpoolprotocols/stratum/EthStratumClient.h

```diff
@@ -31,7 +31,7 @@ class EthStratumClient : public PoolClient
 	void disconnect();
 	
 	// Connected and Connection Statuses
-	bool isConnected() { return m_connected.load(std::memory_order_relaxed); }
+	bool isConnected() { return (m_disconnecting.load(std::memory_order_relaxed) ? false : m_connected.load(std::memory_order_relaxed)); }
```

Comment:
Can be simplified to `!m_disconnecting.load(std::memory_order_relaxed) && m_connected.load(std::memory_order_relaxed)`.

Reply:
I would also swap the order to easier read: `m_connected.load(std::memory_order_relaxed) && !m_disconnecting.load(std::memory_order_relaxed)` unless the order of checking is important.

Reply:
I thought short circuit "if" was better readable for me but I can handle it.